### PR TITLE
cherry pick  blockinfo and swagger  commits from feature/0.9.3

### PR DESCRIPTION
**Pull request by wade-liwei**

### CODE REVIEW

File: backend/service/service_block.go [hunk 1]
```diff
@@ -35,7 +141,7 @@ func (service *BlockService) QueryList(page, size int) model.PageVo {
 	var blocks []document.Block
 
 	sort := desc(document.Block_Field_Height)
-	var cnt, err = pageQuery(document.CollectionNmBlock, selector, nil, sort, page, size, &blocks)
+	var cnt, err = pageQuery(document.CollectionNmBlock, selector, bson.M{"height": bson.M{"$gt": 0}}, sort, page, size, &blocks)
```
Comment (shineabel) [inline backend/service/service_block.go]: 以后的查询语句需加上执行计划结果（数据量说明）

File: backend/lcd/stake.go [hunk 2]
```diff
@@ -47,6 +47,20 @@ func QueryWithdrawAddr(address string) (result string) {
 	return
 }
 
+func GetAllDelegationsByDelegatorAddr(delAddr string) (delegations []DelegationVo) {
```
Comment (zhiqiang-bianjie) [inline backend/lcd/stake.go]: GetAllDelegationsByDelegatorAddr --> GetDelegationsByDelAddr

File: backend/service/service_block.go (hunk 1 above)
Comment (wade-liwei) [reply backend/service/service_block.go]: 好的；

File: backend/types/const.go [hunk 3]
```diff
@@ -39,15 +39,15 @@ const (
...
+	UrlRegisterQueryValidatorSet    = "/stake/validatorset"
 	//Tx
 	UrlRegisterQueryTxList       = "/tx/{type}/{page}/{size}"
 	UrlRegisterQueryRecentTx     = "/txs/recent"
 	UrlRegisterQueryTxsCounter   = "/txs/statistics"
 	UrlRegisterQueryTxsByAccount = "/txsByAddress/{address}/{page}/{size}"
 	UrlRegisterQueryTxsByDay     = "/txsByDay"
 	UrlRegisterQueryTx           = "/tx/{hash}"
-
+	UrlRegisterQueryTokenFlow    = "/tx/token/flow"
```
Comment (shineabel) [inline backend/types/const.go]: /tx/coinflow

File: backend/service/service_block.go [hunk 4]
```diff
@@ -15,15 +20,116 @@ func (service *BlockService) GetModule() Module {
...
+
+	for _, v := range blockRes.Results.BeginBlock.Tags {
+		if v.Key == "mint-coin" {
+			result.Rewards = utils.ParseCoins(v.Value)
+		}
+	}
+
+	result.PropoperAddr = validatorDoc.OperatorAddress
+	result.PropopserMoniker = validatorDoc.Description.Moniker
+	result.LastBlock = height - 1
```
Comment (zhiqiang-bianjie) [inline backend/service/service_block.go]: when height = 1，"lastBlock" is null

**Pull request merged**